www-servers/apache/apache-2.4.66.recipe

```diff
@@ -176,11 +176,6 @@ REQUIRES="
 	lib:libpcre$secondaryArchSuffix
 	lib:libssl$secondaryArchSuffix
 	"
-if [ -n "$secondaryArchSuffix" -a -z "$commandSuffix" ]; then
```

Comment:
@jmairboeck I guess we can drop this by now, pkgman doesn't yield a version for gcc2 anymore.